gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState, useCallback } from "react";
+import ChevronUp from "@nice-digital/icons/lib/ChevronUp";
+
+import styles from "./BackToTop.module.scss";
+
+interface BackToTopProps {
+	scrollTargetId?: string;
+}
+
+export const BackToTop: React.FC<BackToTopProps> = ({
+	scrollTargetId: scrollTo = "content-start",
+}: BackToTopProps) => {
+	const [isFixed, setIsFixed] = useState(false);
+
+	const handleScroll = useCallback(() => {
+		const footer: Element | null = document.querySelector("footer");
+
+		if (!footer) return;
+
+		const footerVisible: boolean =
+			window.scrollY + window.innerHeight >
+			document.body.scrollHeight - footer.clientHeight;
+
+		const scrolledDown: boolean = window.scrollY > 800;
```

Comment:
Wonder if this 800 could either be a constant defined at the top like ScrollThreshold or be 1.5 times the window height or something? Thinking the second makes sense - you have to scroll a screen and a half to get the back to top seems sensible to me. Although just checked [the GDS one](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps) and it's 2 screens height